README.md

```diff
@@ -3,8 +3,14 @@
 
 ### AirDrop for all Platform (Windows, Linux, Mac, Android, iOS)
 
+
+<img src="screenshots/moveit_logo_readme.png" width="400px" height="400px">
```

Comment:
Logo need to be centered, and It's too big for the screen. Can you make it 200px X 200px?

cc: @awesome-ana

Reply:
ok